snapcraft/tests/integration/general/test_build_properties.py

```diff
@@ -51,6 +51,8 @@ def assert_expected_build_state(self, project_dir):
         self.assertThat(state.properties['stage-packages'], Equals(['curl']))
 
     def test_build_with_arch(self):
+        if self.deb_arch == 'armhf':
+            self.skipTest('For now, we just support crosscompile from amd64')
```

Comment:
Perhaps I'm nit-picking here, but this seems a bit wrong. The condition will let the test run on anything that's not armhf.

So either it should be
-  `if self.deb_arch == 'armhf': self.skipTest('For now we're not testing cross-compiling from armhf')`
or
 - `if self.deb_arch == 'amd64': self.skipTest('For now we're only testing cross-compiling from amd64')`

Note that I'd avoid the term "support" here because this is only about testing.

Reply:
Is it only about testing though?